src/bilder/images/edxapp/templates/edxapp/mitxonline/studio_only.yml.tmpl

```diff
@@ -2,6 +2,8 @@
 ADDL_INSTALLED_APPS:  # ADDED KEY
   - git_auto_export
 COURSE_AUTHORING_MICROFRONTEND_URL: /course-authoring
+FEATURES:
+    ENABLE_V2_CERT_DISPLAY_SETTINGS: true # ADDED KEY
```

Comment:
This will need to go in the `common_settings` file because it already has a `FEATURES` dict and is shared between LMS and studio. Adding this in the studio config will generate invalid YAML since it will be redefining the name further down in the YAML file. The way these files are used is that `common` and `studio` templates get concatenated and then the unified template is rendered. Same for `common` and `lms` templates.

Reply:
Ah, @blarghmatey thanks for the clarification. I thought about adding it to common first but wasn't clear about that with something that's only for Studio. Now, this makes sense. I'll change this.

Reply:
I've updated the PR.